### ISSUE TITLE
Chinese Language abbrievation

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/janis#53]
In our project summary, we list the top languages used in Austin:
https://docs.google.com/document/d/1ll72dBQ9bCh9QA5OaDUzjvJyRO-ZvQ5WbPvmIvKexGU/edit#

We're under the impression that in an ideal world, we'll be handling translations for all those listed. When it comes to the two distinct versions of Chinese (Traditional & Simple) we need confirmation that these will both be implemented. And if they are, we need to research how the language codes are different and refactor our i18n selection banner to accept language codes longer than 2 characters.

Relevant discussion w/ @ifsimicoded :
https://github.com/cityofaustin/janis/pull/46/files#r158395784

cc/ @cthibaultatx

**Comment by cthibaultatx** [comment]
@mateoclarke the top 3 languages are: 1) English, 2) Spanish, 3) Vietnamese which takes up the 3 fixed abbreviations that show up until a user chooses a different language. Once someone selects Chinese (traditional), I believe they should see: 中國 (short for 中國傳統的). If they select Chinese (simplified) they would see: 简体 (short for 简体中文). We will need to check these with translators but we can start there.

**Comment by cthibaultatx** [comment]
@mateoclarke now that we're not using abbreviations for MVP, should we backlog this?

**Comment by mateoclarke** [comment]
Yes, since we aren't using abbreviations let's close this out and reopen or start a new issue if we need later.

**Comment by mateoclarke** [comment]
@cthibaultatx I've encountered a new problem with the language codes we're using for Chinese. Should we loop in @ashleecoa & Courtney Morgan as I believe they are working on language access.

What we're going off of is that:
Simplified Chinese uses the code 'zh-hans'
Traditional Chinese uses the code 'zh-hant'

Neither of these code are supported by Firefox. At least not the version I've downloaded.

![screen shot 2018-01-12 at 5 42 47 pm](https://user-images.githubusercontent.com/5697474/34899774-8f664758-f7c0-11e7-85cf-1e04705c7785.png)

It looks like Chrome does have them as options but they are coded as 
zh-TW for Traditional
zh-CN for Simplified

<img width="528" alt="screen shot 2018-01-12 at 5 47 38 pm" src="https://user-images.githubusercontent.com/5697474/34899859-13e2673c-f7c1-11e7-826a-c44a80d7531e.png">

Do we know if Traditional Chinese is just another way to say Chinese from Taiwan?

Are we implementing translations for both for sure?
